internal/cmd/operator-sdk/generate/internal/manifests.go

```diff
@@ -31,23 +31,6 @@ func GetManifestObjects(c *collector.Manifests, extraSAs []string) (objs []clien
 		objs = append(objs, &c.V1beta1CustomResourceDefinitions[i])
 	}
 
-	// All ServiceAccounts passed in should be written.
```

Comment:
Does this do anything useful or was it just broken behavior from the beginning?

Reply:
Broken behavior, based on incorrect assumptions.